index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title></title>
+</head>
+<body>
+  <section id="projects">
```

Comment:
Don't forget to add some kind of class that labels this as a template (and hide it via CSS). You'll probably want to have another element inside of your section (like an article) with a template class so that that particular article is hidden, but your projects (when you've constructed them) can be appended to your projects section and will show up.